src/host/tracing.cpp

```diff
@@ -6,6 +6,19 @@
 #include "../types/UiaTextRangeBase.hpp"
 #include "../types/ScreenInfoUiaProviderBase.h"
 
+TRACELOGGING_DEFINE_PROVIDER(g_hConhostV2EventTraceProvider,
+                             "Microsoft.Windows.Console.Host",
+                             // {fe1ff234-1f09-50a8-d38d-c44fab43e818}
+                             (0xfe1ff234, 0x1f09, 0x50a8, 0xd3, 0x8d, 0xc4, 0x4f, 0xab, 0x43, 0xe8, 0x18),
+                             TraceLoggingOptionMicrosoftTelemetry());
+
+static const auto cleanup = []() {
+    TraceLoggingRegister(g_hConhostV2EventTraceProvider);

```

Comment:
can we not register this in `WinMain()`? Since we're inside `host` and all.

Reply:
We could do that! I assume you would prefer that?

Reply:
feels less "weird", but for no good reason.